services/indexes/models/types.go

```diff
@@ -3,7 +3,14 @@
 
 package models
 
+type CChainType uint16
```

Comment:
Same as the model bit with these types, why not make them part of the existing transaction and output types?